### ISSUE TITLE
Node name inconsistency in ONNX handling ModuleList

### ISSUE THREAD

**Issue by cwhuang888** [pytorch/pytorch#14464]
I have encountered a conversion issue when handling a nested PyTorch model with ModuleList.
Following is a simple snippet of a real case.
Network: Network(
  (stem): Sequential(
    (0): Conv2d(3, 48, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1), bias=False)
    (1): BatchNorm2d(48, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
  )
  (cells): ModuleList(
    (0): Cell(
      (preprocess0): ReLUConvBN(
        (op): Sequential(
          (0): ReLU()
          (1): Conv2d(48, 16, kernel_size=(1, 1), stride=(1, 1), bias=False)
          (2): BatchNorm2d(16, eps=1e-05, momentum=0.1, affine=False, 
           track_running_stats=True)
        )
      ))))
In Python side, the graph node names are correctly obtained via the dictionary keys
'stem.0', 
'stem.1', 
'cells.0.preprocess0.op.0'
'cells.0.preprocess0.op.1', 
'cells.0.preprocess0.op.2',

However, graph nodes obtained from trace returned from torch.jit.get_trace_graph(model, tuple(args)) gives: 
Network/Sequential[stem]/Conv2d[0]
Network/Sequential[stem]/BatchNorm2d[1]
Network/Cell/ReLUConvBN[preprocess0]/Sequential[op]/ReLU[0]
Network/Cell/ReLUConvBN[preprocess0]/Sequential[op]/Conv2d[1]
Network/Cell/ReLUConvBN[preprocess0]/Sequential[op]/BatchNorm2d[2]

Where the last three should be
Network/**ModuleList[cells]/Cell[0]**/ReLUConvBN[preprocess0]/Sequential[op]/ReLU[0]
Network/**ModuleList[cells]/Cell[0]**/ReLUConvBN[preprocess0]/Sequential[op]/Conv2d[1]
Network/**ModuleList[cells]/Cell[0]**/ReLUConvBN[preprocess0]/Sequential[op]/BatchNorm2d[2]

_Originally posted by @cwhuang888 in https://github.com/pytorch/pytorch/issue_comments#issuecomment-442382807_

cc @BowenBao @neginraoof

**Comment by qsundarr** [comment]
I am using PyTorch 1.1 and running in to the same issue described above.  Has this issue been resolved?
For multiple reasons, I am unable to upgrade to PyTorch1.2 or PyTorch1.3. Is there a work around for this issue? Thanks.

**Comment by andyhahaha** [comment]
I have the same issue. This issue also happened when you call nn.sequence one by one as below.
Really need to update this issue. Thanks!

(stem): Sequential(
(0): Conv2d(3, 48, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1), bias=False)
(1): BatchNorm2d(48, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
)

x = stem[0](x)
x = stem[1](x)

**Comment by garymm** [comment]
Closing due to lack of reply from creator. Please open a new issue with clear repro instructions if this is still an issue on latest nightly PyTorch.
